purchase_order_approved/views/purchase_order_view.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2017 Eficent Business and IT Consulting Services S.L.
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+
+    <record id="purchase_order_form" model="ir.ui.view">
```

Comment:
would be nice to have a filter for "to approve" PO in the search view.

Reply:
It is already available. "to approve" state is default in Odoo.
![selection_042](https://user-images.githubusercontent.com/23449160/37518999-e725d11a-2917-11e8-8038-b4d7ff50a11f.png)